tests/jest/BookNavigator/book-navigator.test.js

```diff
@@ -320,10 +320,11 @@ describe('<book-navigator>', () => {
       describe('Shortcuts', () => {
         test('has specific order of menu shortcuts to show', () => {
           const el = fixtureSync(container());
-          expect(el.shortcutOrder[0]).toEqual('fullscreen');
-          expect(el.shortcutOrder[1]).toEqual('volumes');
-          expect(el.shortcutOrder[2]).toEqual('search');
-          expect(el.shortcutOrder[3]).toEqual('bookmarks');
+          expect(el.shortcutOrder[0]).toEqual('chapters');
+          expect(el.shortcutOrder[1]).toEqual('fullscreen');
+          expect(el.shortcutOrder[2]).toEqual('volumes');
+          expect(el.shortcutOrder[3]).toEqual('search');
+          expect(el.shortcutOrder[4]).toEqual('bookmarks');
```

Comment:
Can you check if there's a way we can check the entire array? Maybe `.toBe`?
```suggestion
          expect(el.shortcutOrder).toEqual([
              'chapters',
              ....
         ])
```